src/components/BlogList/BlogList.jsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Box, Typography, Button } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import { CardActionArea } from '@mui/material';
+import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import plant from "../../Images/heroSen.jpg"
+import Footer from 'components/Footer/Footer';
+
+const styles = {
+    card: {
+        boxShadow: "0px 4px 5px 0px rgba(33, 68, 0, 0.50)",
+        width: 400,
+        display: "flex",
+        flexDirection: "column",
+        flex: 1
+    },
+    containerBlog: {
+        flexGrow: 1,
+        marginTop: "100px",
+        width: "100%",
+        padding: "0px 100px"
+    },
+    myBlogTitle: {
+        fontSize: "35px",
+        textAlign: "center",
+        marginBottom: "30px",
+        fontWeight: "500",
+        color: "#214400"
+    }
+}
+
+
+
+function BlogList({ data }) {

```

Comment:
You should read more about the concept of props destructuring in React

Read here: [Destructuring Props](https://handsonreact.com/docs/props#destructuring-props)